package.json

```diff
@@ -79,8 +79,8 @@
         "@louislam/sqlite3": "15.1.6",
         "@vvo/tzdb": "^6.125.0",
         "args-parser": "~1.3.0",
-        "axios": "~0.28.0",
-        "axios-ntlm": "1.3.0",
+        "axios": "~1.6.4",
```

Comment:
Until axios provides a migration guide or documents what (breaking) changes have been made, just bumping the version is too risky for us.

They litterally only state the folling in their [changelog](https://github.com/axios/axios/blob/v1.x/CHANGELOG.md#100---2022-10-04):
> There are multiple deprecations, refactors and fixes provided in this release. Please read through the full release notes to see how this may impact your project and use case.